lib/galaxy/tool_util/client/landing_library.sample.yml

```diff
@@ -0,0 +1,13 @@
+simple_workflow:
```

Comment:
I find the term library (here and elsewhere) a bit confusing. What about:

* controls
* properties
* options
* configurations
??

Reply:
Is "catalog" okay? I think of those choices - options is the best but it is feels a little overloaded with options in the workflow instead of options for which workflow to run.

Reply:
catalog is also nice :)

Reply:
Done!